quipucords/manage.py

```diff
@@ -1,14 +1,3 @@
-#!/usr/bin/env python
```

Comment:
@infinitewarp Could this be the culprit causing issues? We might need this shebang line :thinking:

Reply:
Deleting the shebang here was a mistake. Thanks for catching it!